docs/install_guides/installing-openvino-docker-linux.md

```diff
@@ -2,33 +2,39 @@
 
 The Intel® Distribution of OpenVINO™ toolkit quickly deploys applications and solutions that emulate human vision. Based on Convolutional Neural Networks (CNN), the toolkit extends computer vision (CV) workloads across Intel® hardware, maximizing performance. The Intel® Distribution of OpenVINO™ toolkit includes the Intel® Deep Learning Deployment Toolkit.  
 
-This guide provides the steps for creating a Docker* image with Intel® Distribution of OpenVINO™ toolkit for Linux* and further installation.  
+This guide provides device specific of creation a Docker* image with Intel® Distribution of OpenVINO™ toolkit for Linux* and further usage.  
 
 ## System Requirements
 
 **Target Operating Systems**
 
 - Ubuntu\* 18.04 long-term support (LTS), 64-bit
 - Ubuntu\* 20.04 long-term support (LTS), 64-bit
-- CentOS\* 7.6
-- Red Hat* Enterprise Linux* 8.2 (64 bit)
+- CentOS\* 7
+- Red Hat\* Enterprise Linux* 8 (64 bit)
 
 **Host Operating Systems**
 
-- Linux with installed GPU driver and with Linux kernel supported by GPU driver
+- Linux
 
 ## Prebuilt images
 
-Prebuilt images are available on: 
+Prebuilt images are available on:
+
 - [Docker Hub](https://hub.docker.com/u/openvino)
 - [Red Hat* Quay.io](https://quay.io/organization/openvino)
 - [Red Hat* Ecosystem Catalog](https://catalog.redhat.com/software/containers/intel/openvino-runtime/606ff4d7ecb5241699188fb3)
 
+## Build a Docker* Image
+
+You can use [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles) or generate a Dockerfile with your setting via [DockerHub CI Framework](https://github.com/openvinotoolkit/docker_ci). The Framework can generate a Dockerfile, build, test, and deploy an image with the Intel® Distribution of OpenVINO™ toolkit.
+As well, you can try our [Tutorials](https://github.com/openvinotoolkit/docker_ci/tree/master/docs/tutorials) which demonstrate usage of OpenVINO™ Docker containers.
```

Comment:
As well, you can try our ---> You can also try our
demonstrate usage ---> demonstrate the usage